### PR TITLE
libpciaccess: PCI access utility library

### PR DESCRIPTION
**Pull request by drscream**
The pciaccess library wraps platform-dependent PCI access methods in a convenient library. For exmaple it's required to build gfx-drm and the kernel module.

### CODE REVIEW

**Comment by oetiker** [comment]
make sure you take note of https://github.com/omniosorg/omnios-extra/blob/master/doc/structure.md

**Comment by drscream** [comment]
@oetiker thanks a lot for the hint, I will have a look and update my pull request as recommended. Is the version number suffix mandatory, because it's not used by lot's of other package in the build folder?

**Comment by oetiker** [comment]
Whether a package uses a version number depends on the package and its evolution ... if your package contains a shared library, the naming choice of both the package and the directory will determine your path for upgrading it when new versions come out. If it is a library where multiple versions could conceivably co-exist on the same system, you should take care that this is possible ... 

by incorporating the major version of the library into the package name for example (and the directoy too) ... 

in other words, there is no hard and fast rule but you are required to decide what works best for your package

**Comment by drscream** [comment]
I've changed the `PKG` name based on your request. I hope the following installation is okey for the package:

```
PATH
opt/ooce
opt/ooce/include
opt/ooce/include/pciaccess.h
opt/ooce/lib
opt/ooce/lib/amd64
opt/ooce/lib/amd64/libpciaccess.a
opt/ooce/lib/amd64/libpciaccess.so
opt/ooce/lib/amd64/libpciaccess.so.0
opt/ooce/lib/amd64/libpciaccess.so.0.11.1
opt/ooce/lib/amd64/pkgconfig
opt/ooce/lib/amd64/pkgconfig/pciaccess.pc
opt/ooce/lib/libpciaccess.a
opt/ooce/lib/libpciaccess.so
opt/ooce/lib/libpciaccess.so.0
opt/ooce/lib/libpciaccess.so.0.11.1
opt/ooce/lib/pkgconfig
opt/ooce/lib/pkgconfig/pciaccess.pc
```

**Comment by drscream** [comment]
@hadfl I've fixed the order, sorry for the delay.
